config/locales/views/wiki/revisions/de.yml

```diff
@@ -11,8 +11,7 @@ de:
         <th>#</th>
         <th>Benutzer</th>
         <th>Erstellt</th>
-        <th></th>
-        <th></th>
+        <th>Aktion</th>
```

Comment:
In general unless you're 100% sure about a translation (i'm not, i don't speak german!) we don't have to mess with these because it'll default to English. But since this is a structural change as well, I'll merge this one, thank you!!!!

Reply:
i will keep in mind .Thanks @jywarren